rct229/rules/section22/section22rule5.py

```diff
@@ -0,0 +1,77 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.utils.jsonpath_utils import find_all
+
+APPLICABLE_SYS_TYPES = [
+    "SYS-11.1",
+    "SYS-11.2",
+    "SYS-11.1A",
+    "SYS-11.2A",
+    "SYS-11B",
+    "SYS-11C",
+]
+TEMP_RESET_OPTION = schema_enums["TemperatureResetOptions"]
```

Comment:
I wonder if it is better to call this `TEMPERATURE_RESET` without the `_OPTION`. Since this is an enumeration, the `_OPTION` or similar word is implied.